### ISSUE TITLE
Normalization of mapping tables

### ISSUE THREAD

**Issue by TjarkMiener** [ctlearn-project/ctlearn#85]
The normalization in image mapper isn't working properly. The core problem is the following:
1. Firstly each mapping table is normalized by `normalization(self, mapping_matrix3d, num_pixels)`. For example for the mapping table of the one camera pixel in the figure, the weights of the output grid points (red and green and from more Delaunay triangulation around the pixel) are normalized.
2. Then, the mask is applied by `mask_interpolation(self, mapping_matrix3d, nn_index, num_pixels, pad)`. The mask will set the red grid points to zero and keep the green points. This will somehow destroy the normalization and we will end up having less intensity/energy in the transformed image.

Applying the mask before the normalization will result in the same problem.
![normalization](https://user-images.githubusercontent.com/37835610/51491693-702d2580-1daf-11e9-80ba-5238a0ed9905.png)

**Comment by TjarkMiener** [comment]
I added an optional argument in image_mapper for mask interpolation in commit e1639c1017146139a311e5a0b644a2c3816cc992

**Comment by aribrill** [comment]
When the argument for mask interpolation is set to True, does this issue still persist? If so, I think this issue should be reopened. If I understand correctly, the point of mask interpolation is to prevent contamination from zero padding which causes a border artifact, so it should be set to True by default.

**Comment by TjarkMiener** [comment]
Yes! When mask interpolation is True it's still persist. However, I think this is an issue that occur by construction and can't be avoided.
